spec/job/adapters/linux_host/launcher_spec.rb

```diff
@@ -159,6 +159,22 @@ def content
                 :session_pid=>"175138"
             }
         ] }
+        let(:tmux_broken_output) { "----------------  ------------------------------------
+            Usage Statistics for project PZS0714
+Time              2021-09-12 to 2021-09-13
+PI                alanc@osc.edu
+Remaining Budget  -0.25
+----------------  ------------------------------------
+
+User           Jobs    Dollars    Status
+-------------  ------  ---------  --------
+alanc          0       0.0        ACTIVE
+johrstrom      0       0.0        ACTIVE
+johrstromtest  0       0.0        ACTIVE
+travert        0       0.0        ACTIVE
+--             --      --
+TOTAL          0       0.0
+launched-by-ondemand-a8e85cd4-791d-49fa-8be1-5bd5c1009d70,1569609529,175138" }
```

Comment:
Can we turn this into a HEREDOC, they just look nicer and can be tabbed over to the right indentation.

Plus I think the test can be a little bit tougher. Maybe some lines that have 1 or 2 keys not all 3. And some lines that have the seperator (and use the `"#{seperator}"` here too in case it changes in the future).

Reply:
Added the HEREDOC and test cases for returns with missing keys. The `#{seperator}` is checked already in the tests under the var name of `sep` for the `tmux_output_{a,b,c}` cases.